src/Our.Umbraco.HttpsRedirect/Events/ApplicationEventsHandler.cs

```diff
@@ -12,11 +12,22 @@ namespace Our.Umbraco.HttpsRedirect.Events
 {
     public class EventsHandler : ApplicationEventHandler
     {
-        public EventsHandler()
+
+        /// <summary>
+        /// Register event handler on start.
+        /// </summary>
+        /// <param name="httpApplicationBase">Umbraco application.</param>
+        /// <param name="applicationContext">Application context.</param>
+        protected override void ApplicationStarted(UmbracoApplicationBase umbracoApplication, ApplicationContext applicationContext)

```

Comment:
It looks like ApplicationStarted came in with v6.1.0 per https://our.umbraco.org/documentation/Reference/Events/application-startup